.github/pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-**Please delete this line before submitting**, _Pull request TITLE should look like this_: `[Resource] -> [Resource Section In Docs]`
```

Comment:
please revert the template file change.

Reply:
Done

![image](https://github.com/bradtraversy/design-resources-for-developers/assets/63359966/59de6e71-485a-442f-ac86-f2de7793ad77)